MainActivity.java

```diff
@@ -0,0 +1,169 @@
+package ai.fritz.heartbeat.activities;
```

Comment:
Let's remove this (see comment)

Reply:
I'm having a hard time understanding what you  want me to remove?

Reply:
Removing this class from the top level of fritz-android-tutorials:
![Screen Shot 2019-05-09 at 10 47 28 AM](https://user-images.githubusercontent.com/31963796/57463010-f2026b00-7247-11e9-8857-2298a35b0c1e.png)

You've already modified it here: fritz-demo-app/app/src/main/java/ai/fritz/heartbeat/activities/MainActivity.java

Removing this from your PR makes sure that we don't accidentally add classes that are out of place.